recipe/meta.yaml

```
@@ -1,52 +1,56 @@
 {% set name = "httpstan" %}
-{% set version = "4.6.1" %}
+{% set version = "4.13.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/stan-dev/httpstan/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 703e5e04e60651e0004574bb9695827d759fd13eb0d6bd67f827c1bfa0a1fd31
+  sha256: 6b15a07557715e79e6fd66993930003b270f8b8b0c9e65f84978afe5e6bb3047
 
 build:
-  number: 3
-  script: "{{ PYTHON }} -m pip install . -vv"
-  skip: True  # [py<37] 
-  
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+  skip: True  # [py<310]
+
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - pybind11                               # [build_platform != target_platform]
     - {{ compiler('cxx') }}
   host:
     - pip
-    - poetry  >=1.1.4
-    - pybind11  >=2.6.2
+    - poetry-core >=1.0.0
+    - pybind11  >=2.9.2
     - python
+    - setuptools
   run:
+    - python
     - aiohttp  >=3.7
     - appdirs  >=1.4
-    - lz4  >=3.1
     - marshmallow  >=3.10
-    - numpy  >=1.16
-    - python
-    - setuptools  >=41.0
-    - webargs  >=7.0
+    - numpy  >=1.19
+    - webargs  >=8.0
```

```suggestion
    - webargs  >=8.0
    - setuptools >=48.0
```
`setuptools` is needed at runtime, so I think it should be left in the run section

---

Out of curiosity, I see it under build in pyproject.toml also.
https://github.com/stan-dev/httpstan/blob/2bb63becd5f9a2931ad0bf48ce9e87b81fbf9eac/pyproject.toml#L77

In this case does it matter if it's under one, or the other?

---

It's needed both at build and run, so listing it both places makes sense.

---

See [this](https://github.com/stan-dev/httpstan/blob/adb90820e9d59cf19c43d53b4c8414fbbd8bb712/httpstan/models.py#L20)

---

Got it that makes sense. I completely missed that. Thank you!